featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -120,7 +121,7 @@ def test_init_and_name(es):
     for transform_prim in trans_primitives:
         # skip automated testing if a few special cases
         features_to_use = log_features
-        if transform_prim in [NotEqual, Equal]:
+        if transform_prim in [NotEqual, Equal, FileExtension]:
```

Comment:
Is `FileExtension` here because we don't have any `Filepath` logical type columns in our data?

Reply:
Yes